src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -1,90 +1,123 @@
      * вида Arrays.asList( new People(«Вася», 16, Sex.MAN),
      * new People(«Петя», 23, Sex.MAN),
-     * new People(«Елена», 42, Sex.WOMEN),
+     * new People(«Елена», 42, Sex.WOMAN),
      * new People(«Иван Иванович», 69, Sex.MAN)).
-     * Задача: Выбрать мужчин-военнообязанных (от `fromAge` до `toAge` лет)</p>
+     * Задача: Выбрать мужчин-военнообязанных (от `fromAge` до `toAge` лет)
      **/
     public List<People> manSelectByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getSex().equals(People.Sex.MAN)
+                        && person.getAge() >= fromAge
+                        && person.getAge() <= toAge)
+                .collect(Collectors.toList());
     }
 
     /**
-     * <p>6. Дана коллекция класс People (с полями name — имя, age — возраст, sex — пол),
+     * 6. Дана коллекция класс People (с полями name — имя, age — возраст, sex — пол),
      * вида Arrays.asList( new People(«Вася», 16, Sex.MAN),
      * new People(«Петя», 23, Sex.MAN),
-     * new People(«Елена», 42, Sex.WOMEN),
+     * new People(«Елена», 42, Sex.WOMAN),
      * new People(«Иван Иванович», 69, Sex.MAN)).
      * Задача: Найти всех потенциально работоспособных людей в выборке от fromAge до
      * femaleToAge для женщин, и от fromAge до maleToAge для мужчин
-     * Пример: от 18 лет и учитывая что женщины выходят в 55 лет, а мужчина в 60</p>
-     **/
+     * Пример: от 18 лет и учитывая что женщины выходят в 55 лет, а мужчина в 60
+     *
+     */
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getAge() >= fromAge)
+                .filter(person -> (person.getSex().equals(People.Sex.MAN))
+                        ? person.getAge() <= maleToAge
+                        : person.getAge() <= femaleToAge)
+                .collect(Collectors.toList());
     }
 
     /**
-     * <p>7. Дано коллекцию List of peoples. Класс People (с полями name — имя, age — возраст,
+     * 7. Дано коллекцию List of peoples. Класс People (с полями name — имя, age — возраст,
      * sex — пол, List of Cats -  кошки этого человека).
      * Дано класс Cat (name - имя кошки, age - возраст кошки).
-     * Задача: вивести все имена кошек в которых хозяева это девушки старше 18 лет</p>
+     * Задача: вивести все имена кошек в которых хозяева это девушки старше 18 лет
      **/
     public List<String> getCatsNames(List<People> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getSex().equals(People.Sex.WOMAN)
+                        && person.getAge() >= femaleAge)
```

Comment:
@RomanMinevich Can you replace this with several filters?

Reply:
Done.